test/controllers/graphql_controller_v2_test.rb

```diff
@@ -1,46 +1,41 @@
 require 'test_helper'
 
 class GraphqlControllerV2Test < ActionDispatch::IntegrationTest
-  def setup
```

Comment:
This was a feature removed from `config/features.rb` already, right - so its removal here is more about cleanup than any meaningful change?